tsconfig.json

```diff
@@ -25,5 +25,5 @@
     "allowSyntheticDefaultImports": true
   },
   "exclude": ["node_modules"],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"]
+  "include": ["next-env.d.ts", "additional.d.ts", "**/*.ts", "**/*.tsx"]
```

Comment:
This is an additional change since I saw in Next.js docs this is the preferred way to declare additional types.